package.json

```diff
@@ -13,7 +13,7 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "xo && mocha --timeout 20000"
+    "test": "if-node-version \">=4\" xo && mocha --timeout 20000"
```

Comment:
@sindresorhus wait what? I though it was working as it's been fine on yeoman-generator up to now. That being said, we didn't bump lately - things might've changed.